packages/models-library/src/models_library/clusters.py

```diff
@@ -42,21 +42,21 @@ class ClusterAccessRights(BaseModel):
 
 
 class BaseAuthentication(BaseModel):
-    type: str
+    discriminator_type: str
```

Comment:
Q: is there a rationale for changing this name. I am aware this field is used as discriminator but I wonder whether there is another reason for underline that in the name.